{"uid":"3db940a1c5c4983b","name":"test_get_login_invalid_credentials","fullName":"tests.authentication.test_login#test_get_login_invalid_credentials","historyId":"32134287d0d98132759547a795aa5ea9","time":{"start":1726286539890,"stop":1726286540467,"duration":577},"status":"failed","statusMessage":"AssertionError: Expected status code 401 but got 500. Error message: Nombre de usuario o contraseña incorrecta.","statusTrace":"get_header = {'Content-Type': 'application/json-patch+json', 'accept': '*/*'}\ninvalid_credentials = {'confirmPassword': 'invalid_password', 'password': 'invalid_password', 'username': 'invalid_username'}\n\n    @allure.suite('Log in')\n    @allure.epic('Authentication')\n    @allure.feature('Log in')\n    @allure.story('Login Invalid Credentials')\n    @allure.tag('author: Jeyson')\n    @pytest.mark.functional\n    @pytest.mark.negative\n    def test_get_login_invalid_credentials(get_header, invalid_credentials):\n        url = AuthenticationEndpoints.authenticate()\n        response = RealHomeRequest.post(url, get_header, invalid_credentials)\n>       assert_status_code_unauthorized(response)\n\ntests/authentication/test_login.py:62: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [500]>\n\n    def assert_status_code_unauthorized(response):\n>       assert response.status_code == 401, f\"Expected status code 401 but got {response.status_code}. Error message: {extract_error_message(response)}\"\nE       AssertionError: Expected status code 401 but got 500. Error message: Nombre de usuario o contraseña incorrecta.\n\ncore/assertions/status_code.py:14: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"user_credentials","time":{"start":1726286536457,"stop":1726286536457,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"get_header","time":{"start":1726286536457,"stop":1726286536457,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"invalid_credentials","time":{"start":1726286539889,"stop":1726286539889,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"suite","value":"Log in"},{"name":"tag","value":"author: Jeyson"},{"name":"feature","value":"Log in"},{"name":"epic","value":"Authentication"},{"name":"story","value":"Login Invalid Credentials"},{"name":"tag","value":"negative"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.authentication"},{"name":"host","value":"fv-az654-931"},{"name":"thread","value":"2099-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.authentication.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["negative","functional","author: Jeyson"]},"source":"3db940a1c5c4983b.json","parameterValues":[]}